llvm/lib/Target/AArch64/GISel/AArch64LegalizerInfo.cpp

```diff
@@ -1363,6 +1363,11 @@ AArch64LegalizerInfo::AArch64LegalizerInfo(const AArch64Subtarget &ST)
 
   getActionDefinitionsBuilder({G_SCMP, G_UCMP}).lower();
 
+  getActionDefinitionsBuilder(G_EXTRACT_SUBVECTOR)
+      .legalFor({{v8s8, v16s8}, {v4s16, v8s16}, {v2s32, v4s32}})
+      .widenScalarOrEltToNextPow2(0)
+      .immIdx(0); // Inform verifier imm idx 0 is handled.
```

Comment:
immIdx won't d anything here, just remove it

Reply:
I get verification errors before finding `immIdx`. G_MEMCPY and family also use it.

Reply:
Verification errors? Do you mean the covered rule log?

Reply:
```
 getLegacyLegalizerInfo().computeTables();
  verify(*ST.getInstrInfo());
```
Yes.

Reply:
In the log was `FAIL` which was supposed to be `OK` for G_EXTRACT_SUBVECTOR.

Reply:
I'd consider that to be a bug in the verify